linux_os/guide/system/software/integrity/endpoint_security_software/install_antivirus/rule.yml

```diff
@@ -26,30 +27,23 @@ identifiers:
     cce@rhel7: 27140-3
 
 references:
-    disa: 1239,1668
-    nist: 'SC-28,SI-3'
+    disa: 366,1239,1668
+    nist: SC-28,SI-3,SI-3(1)(ii)
     nist-csf: DE.CM-4,DE.DP-3,PR.DS-1
     srg@rhel6: SRG-OS-00270
-    stigid@rhel6: "000284"
+    stigid@rhel6: RHEL-06-000284
```

Comment:
Should be only "000284" because the "RHEL-06-" prefix is now automatically added by the build system.

Reply:
Thanks! Fixed.